libc/sem.cc

```diff
@@ -18,11 +18,11 @@
 struct indirect_semaphore : std::unique_ptr<semaphore> {
     private:
         int references;
-        bool linked;
+        bool named;
```

Comment:
If you decided to rename this member, please "squash" this patch with the one that introduced the previous name.